apps/bmd/src/components/CandidateContest.tsx

```diff
@@ -99,7 +101,19 @@ function CandidateContest({
   vote,
   updateVote,
 }: Props): JSX.Element {
-  const { userSettings } = useContext(BallotContext);
+  const { electionDefinition, precinctId, userSettings } = useContext(
+    BallotContext
+  );
+  assert(!!userSettings, 'userSettings is required to render CandidateContest');
```

Comment:
No need to coerce to boolean.
```suggestion
  assert(userSettings, 'userSettings is required to render CandidateContest');
```